tests/api_fastapi/execution_api/routes/test_variables.py

```diff
@@ -75,3 +86,60 @@ def test_variable_get_access_denied(self, client):
                 "message": "Task does not have access to variable key1",
             }
         }
+
+
+class TestPostVariable(TestVariableEndpoint):
+    def test_should_create_variable(self, client, session):
+        payload = {"key": "var_create", "value": "{}"}
+        response = client.post(
+            "/execution/variables",
+            json=payload,
+        )
+        assert response.status_code == 200
+        response = client.get(f"/execution/variables/{payload['key']}")
+        assert response.json() == {
+            "key": "var_create",
+            "value": "{}",
+        }
+
+    @pytest.mark.parametrize(
+        "payload, status_code",
+        [
+            pytest.param(
+                {"key": "var_create", "value": "{}", "description": "description"}, 200, id="valid-payload"
+            ),
```

Comment:
I'd say remove this -- we're already testing it via `test_should_create_variable` and having a success case in here complicates the test.